### PR TITLE
Add EzPickle to all envs

### PR DESCRIPTION
**Pull request by ffelten**

### CODE REVIEW

**Comment by LucasAlegre** [comment]
Can I merge this? @ffelten

**Comment by ffelten** [comment]
> Can I merge this? @ffelten 

Waiting for @pseudo-rnd-thoughts 's input for the wrappers. We'll see next week I guess.

File: tests/test_envs.py [hunk 1]
```diff
@@ -1,73 +1,98 @@
...
+
+    data_equivalence(env.reset(), pickled_env.reset())
+
+    action = env.action_space.sample()
+    data_equivalence(env.step(action), pickled_env.step(action))
+    env.close()
+    pickled_env.close()
+
+
 def test_deep_sea_treasure():
```
Comment (pseudo-rnd-thoughts) [inline tests/test_envs.py:21]: You should look into `@pytest.mark.parameterize()`, this would simplify your testing

**Review by pseudo-rnd-thoughts — APPROVED** [review]
This looks good to me. One point would be using keyword arguments over position arguments as it is easier to remember what a value means but it doesn't matter for this.

**Pull request merged**